openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -330,6 +332,7 @@ def _get_user_course_outline_and_processors(course_key: CourseKey,  # lint-amnes
         ('enrollment', EnrollmentOutlineProcessor),
         ('enrollment_track_partitions', EnrollmentTrackPartitionGroupsOutlineProcessor),
         ('cohorts_partitions', CohortPartitionGroupsOutlineProcessor),
+        ('teams_partitions', TeamPartitionGroupsOutlineProcessor),
```

Comment:
whoops, you can ignore this.

By the way @mariajgrimaldi , in case you are interested, I wrote up https://github.com/openedx/edx-platform/issues/34459 as future code improvement. Not something you have to worry about for this PR.